runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -325,6 +340,14 @@ public XWalkViewInternal(Context context) {
         mActivity = (Activity) context;
         mContext = getContext();
 
+        if (getScrollBarStyle() == View.SCROLLBARS_INSIDE_OVERLAY) {
+            setHorizontalScrollBarEnabled(false);
+            setVerticalScrollBarEnabled(false);
+        }
+
+        setFocusable(true);
+        setFocusableInTouchMode(true);
+
```

Comment:
There is a JIRA [XWALK-5815](https://crosswalk-project.org/jira/browse/XWALK-5815), about the fact that XWalkView is SurfaceView in default that is not focusable and then doesn't broadcast keyevent, so customer can't override OnKeyListener to pass onKey event, but there is a workaround (actually I never successful, but customer was!) in stackoverflow just as the same with above two lines:

> setFocusable(true);
setFocusableInTouchMode(true);
requestFocus();

**My point** is that since you add them here, does that means XWALK-5815 works well with nothing additional workaround?